torchgeo/transforms/transforms.py

```diff
@@ -97,3 +98,57 @@ def forward(self, sample: Dict[str, Tensor]) -> Dict[str, Tensor]:
             the unchanged input
         """
         return sample
+
+
+class AugmentationSequential(Module):  # type: ignore[misc]
+    """Wrapper around kornia AugmentationSequential to handle input dicts."""
+
+    def __init__(self, *args: Module, data_keys: List[str]) -> None:
+        """Initialize a new augmentation sequential instance.
+
+        Args:
+            *args: Sequence of kornia augmentations
+            data_keys: List of inputs to augment (e.g. ["image", "mask", "boxes"])
+        """
+        super().__init__()
+        self.data_keys = data_keys
+
+        keys = []
+        for key in data_keys:
+            if key == "image":
+                keys.append("input")
+            elif key == "boxes":
+                keys.append("bbox")
+            else:
+                keys.append(key)
+
+        self.augs = K.AugmentationSequential(*args, data_keys=keys)
+
+    def forward(self, sample: Dict[str, Tensor]) -> Dict[str, Tensor]:
+        """Perform augmentations and update data dict.
+
+        Args:
+            sample: the input
+
+        Returns:
+            the augmented input
+        """
+        # Kornia augmentations require masks to be float
```

Comment:
Why is that?

Reply:
@calebrob6 ran into this as well. For the ops that apply to both images and masks the augmentations raise an error that the inputs should be some type of float. I simply added this to avoid a user having to do this every time they want to use AugmentationSequential. This is a temporary fix though as I'm working on a kornia PR where kornia.AugmentationSequential does the casting under the hood.